Utils/layouts/banner.js

```diff
@@ -0,0 +1,59 @@
+import React from "react"
+import { BannerSection, SubSection, BannerText } from "."
+import { i18n, withTranslation } from "../translation/i18n";
+import {Col, Heading, resolveMedia} from "@moneypensionservice/directories";
+import styled from "styled-components";
+
+import LogoEN from "./LogoEN.jsx"
+import LogoCY from "./LogoCY.jsx"
+
+export const MoneyHelperLogo = styled(Col)`
+  display: flex;
+  flex-direction: row;
+  ${resolveMedia.md`
+    justify-content: flex-end;
+ 
+`};
+  `;
+
+  export const LargeLogo = styled(MoneyHelperLogo)`
+  display: none;
+  ${resolveMedia.md`
+    display: flex;
+   padding-top: 15px;
+ 
+`};
+  `
+  export const InlineLogo = styled(MoneyHelperLogo)`
+  display: block;
+  margin-top: 10px;
+  ${resolveMedia.md`
+    display: none;
+ 
+`};
+  `;
+
+
+const MoneyHelperBanner = ({ t }) => {
+   
+    return (
+        <BannerSection align="stretch" style={{ background: "#0F19A0" }} >
+            <SubSection constrained >
+                <Col>
+                    <Heading level={3} textSize="19px" lineHeight="0.5rem" color="#fff">{t("bannerHeading")}</Heading>
```

Comment:
This heading needs a greater value for line-height as it becomes vertically over-condensed on (very) small viewports. The MAS equivalent is 1.5rem. Also the font-size on MAS is 1.125rem which resolves to 18px though this is maybe less critical.

Reply:
Used 1 to constrain what was inherited